mkosi/archive.py

```diff
@@ -63,6 +66,72 @@ def make_tar(src: Path, dst: Path, *, tools: Path = Path("/"), sandbox: Sequence
         )
 
 
+def make_oci(context: Context, dst: Path) -> None:
+    ca_store = dst / "blobs" / "sha256"
+    ca_store.mkdir(parents=True)
+
+    layer_archive = context.staging / context.config.output_tar
+    with layer_archive.open("rb") as f:
+        layer_digest = hashlib.file_digest(f, "sha256").hexdigest()
```

Comment:
`file_digest` only exists in Python 3.11, I will change this in the final version similar to a loop similar to what is done for the `SHA256SUM` logic

Reply:
We could also backport this but I am not sure how to reference licenses in such cases, instead I moved the existing `hash_file` into utils and also use it here